CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 - Fix issue where `$span` would not be recognized in Advanced Data Scrubbing. ([#781](https://github.com/getsentry/relay/pull/781))
 - Accept big-endian minidumps. ([#789](https://github.com/getsentry/relay/pull/789))
+- Re-implement Upstream request retries & re-authentication. ([#788](https://github.com/getsentry/relay/pull/788))
```

Comment:
I would propose to bring back the old "Features" entry that was removed in https://github.com/getsentry/relay/pull/772:

```suggestion
- Detect network outages and retry sending events instead of dropping them. ([#788](https://github.com/getsentry/relay/pull/788))
```